salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -68,40 +68,66 @@ Create fluent-bit ConfigMap:
                 Remove         logtag
+                # Rename just skipped if dest key already exists
+                Rename         kubernetes_labels_app.kubernetes.io/name app
+                Rename         kubernetes_labels_app app
+                Rename         kubernetes_labels_release release
+                Rename         kubernetes_container_name container
+                Rename         kubernetes_host node
+                Rename         kubernetes_namespace_name namespace
+                Rename         kubernetes_pod_name pod
+                Remove_wildcard kubernetes_
+                # Add unknown for all fields as fluent-bit loki
+                # output complain if "Label_Keys" does not exists
+                Add            app unknown
+                Add            release unknown
+                Add            container unknown
+                Add            node unknown
+                Add            namespace unknown
+                Add            pod unknown
+                Add            stream unknown
             [Output]
-                Name           grafana-loki
-                Match          *
-                Url            http://loki:3100/loki/api/v1/push
-                TenantID       ""
-                BatchWait      1
-                BatchSize      10240
-                Labels         {job="fluent-bit"}
-                RemoveKeys     kubernetes,stream,hostname,unit
-                AutoKubernetesLabels false
-                LabelMapPath   /fluent-bit/etc/labelmap.json
-                LineFormat     json
-                LogLevel       warn
-          labelmap.json: |-
-            {
-              "kubernetes": {
-                "container_name": "container",
-                "host": "node",
-                "labels": {
-                  "app": "app",
-                  "release": "release"
-                },
-                "namespace_name": "namespace",
-                "pod_name": "pod"
-              },
-              "stream": "stream",
-              "hostname": "hostname",
-              "unit": "unit"
-            }
-          parsers.conf: |-
+                Name           loki
+                Match          kube.*
+                Host           loki
+                Port           3100
+                Tenant_ID      ""
+                Labels         job=fluent-bit
+                Label_Keys     $container, $node, $namespace, $pod, $app, $release, $stream
+                Auto_Kubernetes_Labels false
+                Line_Format    json
+                Log_Level      warn
+            [Output]
+                Name           loki
+                Match          host.*
+                Host           loki
+                Port           3100
+                Tenant_ID      ""
+                Labels         job=fluent-bit
+                Label_Keys     $hostname, $unit
+                Line_Format    json
+                Log_Level      warn
```

Comment:
Is this doing anything? Like controlling the log level of the plugin? Seems to be only documented for the `[Server]` section :confused:, from what I can see

Reply:
It does, you can set the Log_Level per "component"